cli/operator/node.go

```diff
@@ -350,39 +361,39 @@ var StartNodeCmd = &cobra.Command{
 		nodeProber := nodeprobe.NewProber(
 			logger,
 			func() {
-				logger.Fatal("ethereum node(s) are either out of sync or down. Ensure the nodes are healthy to resume.")
+				logger.Fatal("Ethereum nodes are either out of sync or down. Make sure the nodes are healthy to resume.")
 			},
 			map[string]nodeprobe.Node{
-				"execution client": executionClient,
+				nodeprobe.ExecutionClientNode: executionClient,
 
 				// Underlying options.Beacon's value implements nodeprobe.StatusChecker.
 				// However, as it uses spec's specssv.BeaconNode interface, avoiding type assertion requires modifications in spec.
 				// If options.Beacon doesn't implement nodeprobe.StatusChecker due to a mistake, this would panic early.
-				"consensus client": consensusClient,
+				nodeprobe.ConsensusClientNode: consensusClient,
 			},
 		)
+		if !usingLocalEvents {
+			eventSyncer := setupEventHandling(
+				cmd.Context(),
+				logger,
+				executionClient,
+				validatorCtrl,
+				metricsReporter,
+				networkConfig,
+				nodeStorage,
+				operatorDataStore,
+				operatorPrivKey,
+				keyManager,
+			)
+			nodeProber.AddNode(nodeprobe.EventSyncerNode, eventSyncer)
+		}
 
 		nodeProber.Start(cmd.Context())
 		nodeProber.Wait()
-		logger.Info("ethereum node(s) are healthy")
 
-		metricsReporter.SSVNodeHealthy()
+		logger.Info("Ethereum nodes are healthy")
 
-		eventSyncer := setupEventHandling(
```

Comment:
`setupEventHandling` is always called here and contains the logic for both on-chain and local events